torchgen/executorch/api/types/signatures.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from typing import List, Optional, Set
+
+from torchgen.api.types import Binding, CType
+from torchgen.model import FunctionSchema, NativeFunction
+
+
+@dataclass(frozen=True)
+class CppSignature:
```

Comment:
Don't reuse the same name, it makes it harder to grep for things. ExecutorchCppSignature.